django_recaptcha/static/django_recaptcha/js/widget_v2_checkbox.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+
+  function main() {
+    for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+      const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
+      if (!widgetUuid) {
+        console.warn("reCAPTCHA widget with missing UUID");
+        break;
+      }
+      console.log(`found reCAPTCHA widget with UUID '${widgetUuid}'`);
+
+      const callbackFunctionName = captchaElement.getAttribute("data-callback");
+      if (!callbackFunctionName) {
+        console.warn(`callback function missing from reCAPTCHA widget with UUID '${widgetUuid}'`);
+        break;
+      }
```

Comment:
It’s unclear to me when / if ever this would happen, considering the attribute is set by the project.

If there was a risk of this being unset, feels like we’d be better of with validation server-side so this shows in server logs?